test/README.md

```diff
@@ -47,3 +46,41 @@ Teardown:
 ```
 $ rake vagrant:teardown
 ```
+
+## Vagrant test environment using Docker Compose
+
+This environment is built from sources with CoreOS running in Virtualbox.
+
+Setup:
+
+```
+test $ vagrant up
+test $ vagrant ssh
+core@localhost $ cd /kontena/test
+core@localhost /kontena/test $ docker-compose run --rm test rake compose:setup
+```
+
+Specs:
+
+```
+core@localhost /kontena/test $ docker-compose run --rm test rake
+```
+
+or
+
+```
+core@localhost /kontena/test $ docker-compose run --rm test rspec spec/
+```
+
+Teardown:
+
+```
+core@localhost /kontena/test $ docker-compose run --rm test rake compose:teardown
+test $ vagrant destroy
```

Comment:
Not sure what you mean. The `docker-compose` is executed within the vagrant machine, the `vagrant destroy` is executed in the host machine.